### PR TITLE
Add element and weapon icon mockup

### PR DESCRIPTION
**Pull request by efrenaguilar95**
Need to update this so if weapon icon is missing, element icon moves up. I also tried getting a little overflow for the icons, but got stuck and didn't have time to figure out for now. Wanna push before I lose what is working. Love this art!

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -16,6 +16,12 @@ function CharacterCard({ costume }: { costume: CostumeInfo }): JSX.Element {
       <a>
         <div className={"card " + "rarity" + costume.stars}>
```
Comment (KeziahMoselle) [inline src/components/Card.tsx:17]: Try using template literals whenever possible

```js
<div className={`card rarity${costume.stars}`}>
```

File: src/components/Card.tsx [hunk 2]
```diff
@@ -16,6 +16,12 @@ function CharacterCard({ costume }: { costume: CostumeInfo }): JSX.Element {
...
           <div className="card-art">
+            <div className="element">
+              <img src="\icons\elements\dark.png"></img>
```
Comment (KeziahMoselle) [inline src/components/Card.tsx:20]: Is there a reason you are using \ instead of / ?

**Review by KeziahMoselle — CHANGES_REQUESTED** [review]

File: src/components/Card.tsx (hunk 2 above)
Comment (efrenaguilar95) [reply src/components/Card.tsx:20]: nope, good catch. surprised that even worked

Comment (KeziahMoselle) [reply src/components/Card.tsx:20]: Yeah i'm surprised too! I believe it's Microsoft's way to handle paths so maybe it is handled by Node.js by default

File: src/components/Card.tsx (hunk 1 above)
Comment (efrenaguilar95) [reply src/components/Card.tsx]: will do, ty!

**Review by KeziahMoselle — APPROVED** [review]

**Pull request merged**